src/deploy/extensions/planner.ts

```diff
@@ -45,6 +45,10 @@ export interface ManifestInstanceSpec extends InstanceSpec {
  */
 export interface DeploymentInstanceSpec extends InstanceSpec {
   params: Record<string, string>;
+  allowedEventTypes?: string[];
+  eventarcChannel?: string;
+  extensionVersion?: extensionsApi.ExtensionVersion;
```

Comment:
extensionVersion and extension are already defined on InstanceSpec, which this type extends, so no need to redefine them here

Reply:
Good point, fixed